packages/components/table/src/table-body/styles-helper.ts

```diff
@@ -68,11 +69,19 @@ function useStyles<T extends DefaultRow>(props: Partial<TableBodyProps<T>>) {
         column,
       })
     }
-    const fixedStyle = getFixedColumnOffset(
-      columnIndex,
-      props?.fixed,
-      props.store
-    )
+
+    const fn = parent?.props.spanMethod
+    const fixedStyle = isFunction(fn)
+      ? getFixedColumnOffsetWithSpan(
+          columnIndex,
+          column.fixed,
+          props.store,
+          row,
+          rowIndex,
+          getSpan
+        )
+      : getFixedColumnOffset(columnIndex, column.fixed, props.store)
+
```

Comment:
Can the `getCellStyle` function also be optimized using `colspanIndex`, similar to the `getColspanRealWidth` function?